migration/20200930-re-generate-marc-856-u-urls.sql

```diff
@@ -0,0 +1,10 @@
+-- Delete all generate-marc workcoveragerecords records, so that new links will be created.
+delete from workcoveragerecords where operation = 'generate-marc';
+
+-- Delete cached MARC delta files (those with a start time),
+-- as they will contain invalid links in the 856|u.
+delete from cachedmarcfiles where start_time is not null;
```

Comment:
Is there any reason we need to keep the delta files? Or can we just remove them and start fresh with new full dumps? If the former -- keep the deltas -- then we won't use the second statement (line 6). If the latter, we don't need to delete the `generate-marc workcoveragerecords` (line 2) because these records affect only the generation of deltas.

@jonathangreen Do you have an opinion on this?

Reply:
I think we can just delete the old delta files. I don't see any reason to keep them around, and they are kind of misleading since they will have bad URLs in them.